spec/models/bridge_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Bridge, type: :model do
+  it 'saves' do
+    bridge = Bridge.create(name: 'My First Bridge', payload: '', inboundURL: 'https://bridgeapi.dev/b1234/inbound', outboundURL: 'https://wowservice.io/new/23847923864', method: 'post', retries: 5, delay: 15)
+    expect(Bridge.first).to eq bridge
+  end
+
+  it 'has many env vars' do 
+    bridge = Bridge.create(name: 'My First Bridge', payload: '', inboundURL: 'https://bridgeapi.dev/b1234/inbound', outboundURL: 'https://wowservice.io/new/23847923864', method: 'post', retries: 5, delay: 15)
+
+    bridge.env_vars << EnvVar.create(key: 'database', value: 'a102345ij2')
+    bridge.env_vars << EnvVar.create(key: 'database_password', value: 'supersecretpasswordwow')
+
+    expect(bridge.env_vars.count).to eq 2
+  end
+
+  it 'has many headers' do 
+    bridge = Bridge.create(name: 'My First Bridge', payload: '', inboundURL: 'https://bridgeapi.dev/b1234/inbound', outboundURL: 'https://wowservice.io/new/23847923864', method: 'post', retries: 5, delay: 15)
+
+    bridge.headers << Header.create(key: 'X_API_KEY', value: 'ooosecrets')
+    bridge.headers << Header.create(key: 'Authentication', value: 'Bearer 1oij2oubviu3498')
+
+    expect(bridge.headers.count).to eq 2
+  end
+
+  it 'has many events' do 
```

Comment:
I don't really see what you are testing in this file, TBH. That you can create objects and pass them into an array?